airbyte-integrations/connectors/destination-gcs/src/main/resources/spec.json

```diff
@@ -0,0 +1,328 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "documentationUrl": "https://docs.airbyte.io/integrations/destinations/gcs",
+  "supportsIncremental": true,
+  "supportsNormalization": false,
+  "supportsDBT": false,
+  "supported_destination_sync_modes": ["overwrite", "append"],
+  "connectionSpecification": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "GCS Destination Spec",
+    "type": "object",
+    "required": [
+      "gcs_bucket_name",
+      "gcs_bucket_path",
+      "gcs_bucket_region",
+      "credential",
+      "format"
+    ],
+    "additionalProperties": false,
+    "properties": {
+      "gcs_bucket_name": {
+        "title": "GCS Bucket Name",
+        "type": "string",
+        "description": "The name of the GCS bucket.",
+        "examples": ["airbyte_sync"]
+      },
+      "gcs_bucket_path": {
+        "description": "Directory under the GCS bucket where data will be written.",
+        "type": "string",
+        "examples": ["data_sync/test"]
+      },
+      "gcs_bucket_region": {
+        "title": "GCS Bucket Region",
+        "type": "string",
+        "default": "",
+        "description": "The region of the GCS bucket.",
+        "enum": [
+          "",
+          "-- North America --",
```

Comment:
should these enum values be here? they should not be selectable

Reply:
This seems a nice feature actually. This enum list is pretty long, and the separators help people navigate through it I think.